com/amazonaws/athena/connectors/udfs/SkyflowUDFHandler.java

```diff
@@ -0,0 +1,129 @@
+/*-
+ * #%L
+ * athena-udfs
+ * %%
+ * Copyright (C) 2019 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.udfs;
+
+import com.amazonaws.athena.connector.lambda.handlers.UserDefinedFunctionHandler;
+import com.amazonaws.athena.connector.lambda.security.CachableSecretsManager;
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerClient;
+import com.skyflow.common.utils.HttpUtility;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SkyflowUDFHandler
+        extends UserDefinedFunctionHandler
+{
+    private static final String SOURCE_TYPE = "athena_common_udfs";
+    private final CachableSecretsManager cachableSecretsManager;
+    Boolean logLevel = Boolean.valueOf(System.getenv("LOG_LEVEL"));
+
+    public SkyflowUDFHandler()
+    {
+        this(new CachableSecretsManager(AWSSecretsManagerClient.builder().build()));
+    }
+    
+    SkyflowUDFHandler(CachableSecretsManager cachableSecretsManager)
+    {
+        super(SOURCE_TYPE);
+        this.cachableSecretsManager = cachableSecretsManager;
+    }
+   public String detokenize(String input, String role, String redactionParam)
+    {
+        String response = null;
+        try {
+            String vaultID = System.getenv("VAULT_ID");
+            String vaultURL = System.getenv("VAULT_URL");
+            String secretName = System.getenv("SECRET");
+            String secretString = cachableSecretsManager.getSecret(secretName);
```

Comment:
@skyflow-srivyshnavi just curious, is this a Skyflow secret (e.g. API key) or some AWS secret?